app/controllers/qualifications/feedbacks_controller.rb

```diff
@@ -0,0 +1,29 @@
+class FeedbacksController < ApplicationController
```

Comment:
Here we need to inherit from the `QualificationsInterfaceController`, the parent controller for most routes accessed via the AYTQ service.